core/src/main/scala/cromwell/core/io/IoCommand.scala

```diff
@@ -21,21 +25,46 @@ object IoCommand {
   def defaultBackoff = SimpleExponentialBackoff(defaultGoogleBackoff)
   
   type RetryCommand[T] = (FiniteDuration, IoCommand[T])
+
+  private val logger = LoggerFactory.getLogger(IoCommand.getClass)
 }
 
 trait IoCommand[+T] {
+
+  private val uuid = UUID.randomUUID().toString
+  private val creation = OffsetDateTime.now
+
+  def customDebug(message: String): Unit = {
+    val now = OffsetDateTime.now
+    val millis = java.time.Duration.between(creation, now).toMillis
+    val seconds = millis / 1000D
+    IoCommand.logger.info(f"($uuid) ($seconds%,.3f seconds) $message ($getClass) (thread '${Thread.currentThread().getName}')")
+    if (seconds > 5 * 60) {
+      IoCommand.logger.warn(f"($uuid) ($seconds%,.3f seconds) FYI! `$message` IO IS OVER 5 MINUTES ($getClass) (thread '${Thread.currentThread().getName}')")
```

Comment:
This message may be a little clunky but 🤷

Reply:
Happy to have better english. I just want something I can grep for quickly in the logs, then backtrack from there using the `uuid`.